### DISCUSSION TITLE
Using WebRTC Converted on Server A on Server B

### DISCUSSION THREAD

**Discussion by CheSauRie** [bluenviron/mediamtx#3897] [Questions]
### Question

Hello everyone,

I currently have the following setup:

Server A: Running MediaMTX, where I convert an RTSP stream from a camera into a WebRTC stream.
Server B: Also running MediaMTX, and I would like to reuse the WebRTC stream that was converted on Server A.

How can I use the WebRTC stream that was converted on Server A on Server B? Do I need any specific configuration to pass the WebRTC stream between these two servers?

Thanks in advance for your help!

**Comment by aler9** [comment]
Hello, you can forward the stream from server A to server B by following the guide in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#forward-streams-to-other-servers

It's not necessary to use the WebRTC protocol for forwarding the stream between servers, you can use any protocol you like (the README suggest RTSP for simplicity reasons). Users will still be able to watch the stream, from server B, with WebRTC, since conversion between protocols is automatic in MediaMTX.
